pdns/rec-lua-conf.cc

```diff
@@ -322,11 +322,33 @@ static void rpzPrimary(LuaConfigItems& lci, luaConfigDelayedThreads& delayedThre
   delayedThreads.rpzPrimaryThreads.push_back(std::make_tuple(primaries, defpol, defpolOverrideLocal, maxTTL, zoneIdx, tt, maxReceivedXFRMBytes, localAddress, axfrTimeout, refresh, sr, dumpFile));
 }
 
+// A wrapper clas that load the snadrad Lua defintions into the conext, so that we can use things
+// like pdns.A
+class RecLuaConfigContext : public BaseLua4
+{
+public:
+  RecLuaConfigContext()
+  {
+    prepareContext();
+  }
+  virtual void postPrepareContext() override
+  {
+  }
+  void postLoad() override
+  {
+  }
+  std::unique_ptr<LuaContext>& operator()()
```

Comment:
Perhaps overriding the `->` operator and returning a `LuaContext&` would have been easier to use?